lib/rules/alpha-value-notation/index.mjs

```diff
@@ -99,9 +101,18 @@ const rule = (primary, secondaryOptions) => {
 				const unfixed = value;
 				const index = declarationValueIndex(decl) + alpha.sourceIndex;
 				const endIndex = index + alpha.value.length;
-				const fix = () => {
-					alpha.value = fixed;
-					setDeclarationValue(decl, parsedValue.toString());
+
+				/** @type {Problem['fix']} */
+				const fix = (apply) => {
```

Comment:
questions
minor
@romainmenke @ybiquitous

1.
I find it annoying to have to add the type for every `const fix` declaration.
Can it be avoided?
Maybe having to import the `Problem` type for all fixable rules is not such a big deal?

2.
Are we sure we want the default to be `apply`?
i.e. it could also be `doNotApply`

Reply:
> I find it annoying that you have to add the type for every single `const fix` declaration.
> Can it be avoided?

Inlining the `fix` variable is a way. Otherwise, I think some type annotations are necessary.

For other questions about the function signature, we're discussing it on #7784. Let's continue there.